### DISCUSSION TITLE
Html5 sound on iOS safari 15.*

### DISCUSSION THREAD

**Discussion by hansagames** [goldfire/howler.js#1596] [Q&A]
Hi,
noticed strange behaviour on iOS Safari 15.* (could not reproduce on 14.*).
I create html5 sound with loop true. And for iOS safari browsers sound stops time to time (could not reproduce on other devices).
After investigation found out that audio tag gets pause event, but there are no howler pause event triggered so I assume that its paused by browser, or by internal howler functions that dont trigger event.

More strange it starts to play after some time, and then get paused again and so on. Sometimes it play multiple loops before gets paused, sometime sit happens after 1-3 seconds. Resume most of time happens when end even from howler is triggered (could be because it trigger play again)

Anyone else noticed such behaviour? 

For reference:

```
        this.howl = new Howl({
            src: "src",
            html5: true,
            autoplay: false,
            preload: true,
            loop: true,
        });
```

Thank you

**Comment by jmaldon1** [comment]
Yup this is happening for me, anyone have a fix? It seems to also happen on chrome too. Maybe the problem is iOS in general and not a specific browser.

I tested the same thing on a google pixel and it works fine.

**Reply by jmaldon1** [reply]
My use case is a timer and when the timer ends, an audio file loops 5 times. I use a custom ‘onend’ code that will set ‘loop’ to false when the audio has been played 5 times.

On iOS the audio seems to pretty consistently play 3 times and skips every other play.

**Reply by jmaldon1** [reply]
https://github.com/goldfire/howler.js/issues/1586#issue-1166772007 similiar issue
